app/chapters/[slug]/transacting/transacting-2/page.tsx

```diff
@@ -57,6 +59,7 @@ export default function Genesispt2() {
             className="genesis-p2"
             dangerouslySetInnerHTML={{ __html: genesis.body.html }}
           ></div>
+          <CopyableCode code={'echo scriptPubKeyHex | xxd -r -p'} language='shell' />
```

Comment:
Note that I've renamed `scriptSigHex` to `scriptPubKeyHex` in this challenge as this was incorrect